lib/driver.js

```diff
@@ -435,9 +435,14 @@ class XCUITestDriver extends BaseDriver {
         await quitAndUninstall(errorMsg);
       }
 
-      this.opts.preventWDAAttachments = !util.hasValue(this.opts.preventWDAAttachments) || this.opts.preventWDAAttachments;
-      await adjustWDAAttachmentsPermissions(this.wda, this.opts.preventWDAAttachments ? '555' : '755');
-      this.logEvent('wdaPermsAdjusted');
+      if (!util.hasValue(this.opts.preventWDAAttachments)) {
+        // XCTest prior to Xcode 9 SDK has no native way to disable attachments
+        this.opts.preventWDAAttachments = this.xcodeVersion.major < 9;
```

Comment:
Since the user requested the WDA attachments be disabled, perhaps a little note in the logs to say that it is done on the WDA level and skipped?

Reply:
Add logs